entities/src/api_req_params.rs

```diff
@@ -150,6 +181,17 @@ pub struct GetAssetsByOwner {
     pub cursor: Option<String>,
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize, JsonSchema)]
+#[serde(deny_unknown_fields, rename_all = "camelCase")]
+pub struct GetNftEditions {
+    pub mint: String,
```

Comment:
Should not it be `Pubkey`? Or serde cannot automatically convert this type?

Reply:
in other places we also use string

Reply:
like here? 

```
let mint_address = Pubkey::from_str(&payload.mint)
```

Reply:
GetAssetsByOwner, GetAsset, GetAssetBatch, GetAssetProof, GetAssetProofBatch, GetAssetsByCreator, GetAssetsByAuthority, GetGrouping and others in this file. 

I wouldn't mind making this improvement, but it would require changing the current approach as a whole, which would result in a big change.
Perhaps it would be worth making a separate task to make the changes for all objects, and then checking everything after that.

Reply:
https://linear.app/mplx/issue/MTG-1441/implement-a-custom-serde-serializerdeserializer-to-automatically